package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bigcommerce-cornerstone",
-  "version": "4.4.1",
+  "version": "4.5.0",
```

Comment:
@PeteyRev thank you for your contribution. Can you please rebase and drop the `package-lock.json` change. We do need to fix it and we are going to fix our release script to make sure this does not happen again.

Reply:
@junedkazi Should be gtg